libs/etterlatte-ktor/src/main/kotlin/RouteUtils.kt

```diff
@@ -122,7 +121,7 @@ suspend inline fun PipelineContext<*, ApplicationCall>.withFoedselsnummer(
             if (harTilgangTilPerson) {
                 onSuccess(foedselsnummer)
             } else {
-                logger.info("Har ikke tilgang til person")
+                application.log.info("Har ikke tilgang til person")
```

Comment:
Hadde vært nice å ha en egen logger for denne filen, f.eks. `val logger = LoggerFactory.getLogger("TilgangsSjekk")` (eller routeutils), siden `application.log` gir oss ikke den beste infoen om hvor det faktisk logges fra

Reply:
egen pr for det